tests/api.c

```diff
@@ -58062,6 +58110,7 @@ TEST_CASE testCases[] = {
     TEST_DECL(test_wolfSSL_i2d_PrivateKey),
     TEST_DECL(test_wolfSSL_OCSP_id_get0_info),
     TEST_DECL(test_wolfSSL_i2d_OCSP_CERTID),
+    TEST_DECL(test_wolfSSL_d2i_OCSP_CERTID),
```

Comment:
wolfSSL_X509_REVOKED_get_revocationDate needs a test, too.

Reply:
This will be done in a follow-up PR when new OpenSSL compliant API's are added.